SimPEG/utils/io_utils/io_utils_electromagnetics.py

```diff
@@ -437,7 +437,7 @@ def read_dcip3d_ubc(file_name, data_type):
     is_pole_rx = False
 
     # IP data for dcip3d has a line with a flag we can remove.
-    if data_type != "volt":
+    if obsfile[0][0:6] == 'IPTYPE':
```

Comment:
consider changing to:
 
`if "IPTYPE" in obsfile[0]`

It will be more robust to possible blank spaces